Ginger/GingerCoreNET/DataSource/ActDSTableElement.cs

```diff
@@ -80,6 +82,17 @@ public override void Execute()
                     DataSource = ds;
                 }
             }
+            ObservableList<DataSourceTable> dstTables = DataSource.GetTablesList();
```

Comment:
Move this code to relevant add Row case as it is not needed for other operations for now